cav_msgs/msg/TrajectoryExecutionStatus.msg

```diff
@@ -0,0 +1,16 @@
+# TrajectoryExecutionStatus.msg
```

Comment:
I think it would be good if we could report the current trajectory following error. But that should probably be handled by a different component from the plugin